packages/server/src/plugin/landingPage/default/types.ts

```diff
@@ -172,7 +181,7 @@ type EmbeddableExplorerOptions = {
    *
    * The default value is false.
    */
-  persistExplorerState: boolean;
+  persistExplorerState?: boolean;
```

Comment:
Not seeing a related code change but I do see this is already optional in the `getEmbeddedExplorerHTML` fn so at runtime this change should be a no-op?

But this allows users specifying an `embed` config to leave it out, where they currently can't?

Reply:
yes, this was just the incorrect type previously